acceptance-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+python manage.py migrate
+python manage.py runscript initial_data
+python manage.py runscript test_data
+lsof -i tcp:9500| awk 'NR!=1 {print $2}' | xargs kill
+python manage.py runserver 9500 < /dev/null &
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2893/commits/b827531b3f32ec4b926c108ff17a0db7e5f54ca1